themes/bootstrap3/templates/myresearch/bulk-action-buttons.phtml

```diff
@@ -4,10 +4,23 @@
 <?php endif; ?>
 <?php $user = $this->auth()->isLoggedIn(); ?>
 <nav class="bulkActionButtons">
-  <div class="bulk-checkbox">
-    <input type="checkbox" name="selectAll" class="checkbox-select-all" id="myresearchCheckAll">
-    <label for="myresearchCheckAll"><?=$this->transEsc('select_all_on_page')?></label>
-  </div>
+  <?php
+  $socialConfig = $this->config()->get('config')['Social'];
+  $multiPageSelection = $socialConfig['multi_page_favorites_selection'] ?? false;
+  $checkboxSelectAllType = $socialConfig['checkbox_select_all_favorites_type'] ?? 'none';
```

Comment:
Typically, the default values for settings match the default configurations in config.ini -- but here, the config.ini defaults are different from the fallbacks if those settings are commented out. It might be good to either align the defaults in both places, or else to add a note in the config.ini comments indicating what the defaults are if the settings are omitted.

Reply:
Changed the fallback values to match the default values in config.ini